Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -411,16 +439,31 @@ def refresh_session(host_id: str) -> Dict:
     return response
 
 
-def run_batch_read_cmd(host_ids: list, command_type: str, full_command: str) -> Dict:
+def batch_refresh_session(batch_id: str) -> None:
+    """
+        Batch refresh a RTR session on multiple hosts.
+        :param batch_id:  Batch ID to execute the command on.
+    """
+    demisto.debug('Starting session refresh')
+    endpoint_url = '/real-time-response/combined/batch-refresh-session/v1'
+
+    body = json.dumps({
```

Comment:
Small comment: you can use the `json` option of http_request and pass the dict directlry instead of doing json.dumps.

Reply:
yea just wanted to stay consistent with the rest of the requests in the integration

Reply:
ok. no biggy